docs/DeveloperGuide.md

```diff
@@ -1482,12 +1487,14 @@ maintainability of our code. We went the extra mile for passion and our own lear
 ---------------------------------------------------------------------------------------------------------------------
 ## **Appendix: Planned Enhancements**
 
```

Comment:
<img width="1314" alt="Screenshot 2024-11-11 at 8 51 37 PM" src="https://github.com/user-attachments/assets/fbcca0dd-21e4-47d4-89d5-fd0596c76f7e">
I think for this part, we need to add in the team size as well

Reply:
ok good catch